packages/react-components/src/localization/locales/index.ts

```diff
@@ -3,54 +3,54 @@
 
 import { PartialDeep } from 'type-fest';
 import { ComponentLocale, ComponentStrings } from '..';
-import _en_US from './en-US/strings.json';
-import _en_GB from './en-GB/strings.json';
-import _de_DE from './de-DE/strings.json';
-import _es_ES from './es-ES/strings.json';
-import _fr_FR from './fr-FR/strings.json';
-import _it_IT from './it-IT/strings.json';
-import _ja_JP from './ja-JP/strings.json';
-import _ko_KR from './ko-KR/strings.json';
-import _nl_NL from './nl-NL/strings.json';
-import _pt_BR from './pt-BR/strings.json';
-import _ru_RU from './ru-RU/strings.json';
-import _tr_TR from './tr-TR/strings.json';
-import _zh_CN from './zh-CN/strings.json';
-import _zh_TW from './zh-TW/strings.json';
+import en_US from './en-US/strings.json';
+import en_GB from './en-GB/strings.json';
+import de_DE from './de-DE/strings.json';
+import es_ES from './es-ES/strings.json';
+import fr_FR from './fr-FR/strings.json';
+import it_IT from './it-IT/strings.json';
+import ja_JP from './ja-JP/strings.json';
+import ko_KR from './ko-KR/strings.json';
+import nl_NL from './nl-NL/strings.json';
+import pt_BR from './pt-BR/strings.json';
+import ru_RU from './ru-RU/strings.json';
+import tr_TR from './tr-TR/strings.json';
+import zh_CN from './zh-CN/strings.json';
+import zh_TW from './zh-TW/strings.json';
 
 const createComponentStrings = (localizedStrings: PartialDeep<ComponentStrings>): ComponentStrings => {
-  const strings: ComponentStrings = { ..._en_US };
+  const strings: ComponentStrings = { ...en_US };

```

Comment:
Tangential to this PR, but why do we fallback to `en_US` here? It's a good thing if `tsc` complains when a string is missing in some locale -- ensures that missing localization strings are caught as build errors.

Reply:
I wanted it to be easier for someone to create a PR with a localized string such that they only need to add the string to en_US strings otherwise there would be build errors. The touchdown pipeline would then handle the translation of that added string after the PR has been merged to main. The TAT on new strings from Touchdown is 5 days so I figure it be best to separate the process.